docker/cajtck.sh

```diff
@@ -55,8 +55,18 @@ echo "TS_HOME $TS_HOME"
 chmod -R 777 $TS_HOME
 cd $TS_HOME/bin
 
+if [[ "$JDK" == "JDK11" || "$JDK" == "jdk11" ]];then
+  export JAVA_HOME=${JDK11_HOME}
+  export PATH=$JAVA_HOME/bin:$PATH
+  cp ts.jte.jdk11 ts.jte
+else
+  sed -i "s#^endorsed.dirs=.*#endorsed.dirs=$TCK_HOME/$GF_TOPLEVEL_DIR/glassfish/modules/endorsed#g" ts.jte
```

Comment:
Looks good, if we get class not found errors with JDK11 that will be a hint that we should see if jars in endorsed folder might need to be on the classpath.  If no errors occur, even better.

Reply:
Right.